teuthology/lock/query.py

```diff
@@ -51,7 +51,7 @@ def is_vm(name=None, status=None):
     return status.get('is_vm', False)
 
 
-def list_locks(keyed_by_name=False, **kwargs):
+def list_locks(keyed_by_name=False, tries=10, **kwargs):
```

Comment:
@zmc ...so, please, tell me why there is the _tries_ parameter if it is unused? Also giving an endless loop makes me frustrating when I run `teuthology_suite` to without lock server configured, and given that timeout is zero... I am experiencing trouble to find point where the error occurred.

Reply:
The `tries` parameter is not unused.

I don't understand the issue you're describing, but this is really not the place to report or discuss it. A tracker ticket perhaps?

Reply:
@zmc tracker ticket looks overkill for me for sharing some thoughts, full of formalism and bureaucracy :-)

Reply:
> The `tries` parameter is not unused.

Look carefully, there is no usage of `tries` value in `list_locks`.